lib/generators/rspec/install/templates/spec/rails_helper.rb

```diff
@@ -42,6 +42,13 @@
   # instead of true.
   config.use_transactional_fixtures = true
 
+  # You can uncomment this line to turn off active_record support entirely.
+  # config.use_active_record = false
+
+<% else -%>
+  # Remove this line to enable support for active_record
```

Comment:
WDYT of mentioning `config.use_transactional_fixtures` and `config.fixture_path` here to get full power out of ActiveRecord?

Reply:
Yeah maybe, mostly just thinking aloud with this, it fails a spec which I'm not sure is this PR or would happen for both...

Reply:
Edited.